zipkin-web/src/main/resources/templates/v2/index.mustache

```diff
@@ -2,19 +2,11 @@
   <form class="form-inline" role="form">
     <div class="form-group">
       <select data-placeholder="Service Name" class="form-control input-sm" style="width: 234px;" name="serviceName" id="serviceName">
-        <option value=""></option>
```

Comment:
This thread is about #718, so let's keep it contained there. There's been overwhelming support to undo the server-side mustache templates and I don't want to halt this PR on account of an old issue